bin/cylc-validate

```diff
@@ -71,22 +74,36 @@ def main():
     profiler = Profiler(options.profile_mode)
     profiler.start()
 
+    if not cylc.flags.debug:
+        # for readability omit timestamps from logging unless in debug mode
+        for handler in LOG.handlers:
+            handler.setFormatter(CylcLogFormatter(timestamp=False))
+
     suite, suiterc = SuiteSrvFilesManager().parse_suite_arg(options, args[0])
     cfg = SuiteConfig(
         suite, suiterc,
         load_template_vars(options.templatevars, options.templatevars_file),
         cli_initial_point_string=options.icp,
         is_validate=True, strict=options.strict, run_mode=options.run_mode,
         output_fname=options.output, mem_log_func=profiler.log_memory)
+
     # Check bounds of sequences
-    for seq in cfg.sequences:
-        if seq.get_first_point(cfg.start_point) is None:
-            mesg = '%s: sequence out of bound for initial cycle point %s' % (
-                seq, cfg.start_point)
-            if options.strict:
-                raise SuiteConfigError(mesg)
-            elif cylc.flags.verbose:
-                sys.stderr.write(' + %s\n' % mesg)
+    out_of_bounds = [str(seq) for seq in cfg.sequences
+                     if seq.get_first_point(cfg.start_point) is None]
+    if out_of_bounds:
+        if len(out_of_bounds) > 1:
+            # avoid spamming users with multiple warnings
+            msg = ('multiple sequences out of bound for initial cycle point '
+                   '%s:\n%s' % (
+                       cfg.start_point,
+                       '\n'.join(textwrap.wrap(', '.join(out_of_bounds), 70))))
+        else:
+            msg = '%s: sequence out of bound for initial cycle point %s' % (
```

Comment:
Minor niggle, and it comes from the existing code, but isn't the expression "out of bound"? (plural)